README.md

```diff
@@ -84,9 +82,100 @@ But you'll still need to pick the built version (which ships with the package).
 
 ```javascript
 // with the classic require...
-let Kuzzle = require('kuzzle-sdk/dist/kuzzle.js')
+const { Kuzzle } = require('kuzzle-sdk/dist/kuzzle.js')
 // ... or with the new import directive.
-import Kuzzle from 'kuzzle-sdk/dist/kuzzle.js'
+import { Kuzzle } from 'kuzzle-sdk/dist/kuzzle.js'
+```
+
+## Protocols used
+Actuellement, le SDK supporte nativement 3 protocols: `http`, `websocket` et `socketio`.  
+
+The main reason behind this is that while Socket.IO offers better compatibility with older web browsers, our raw WebSocket implementation is about 20% faster
+
+Which protocol is used when you connect to Kuzzle depends on multiple factors:
+
+#### NodeJS
+
+The protocol used is always raw WebSocket for `websocket`.
+
+#### Web Browsers
+
+If you choose `websocket`, the SDK will first try to use raw WebSocket to connect to Kuzzle. If the web browser does not support this protocol, then the SDK falls back to Socket.IO.
+
+## SDK Documentation
+
+### Connection to Kuzzle
+
+By default, the SDK is not connected to Kuzzle when it is instantiated. You must manually call the `kuzzle.connect()` method before using the SDK.  
+
+It is then possible to interact with the Kuzzle API through the SDK once the Promise returned by `kuzzle.connect()` has been resolved.  
+
+```js
+// Without async/await
+kuzzle.connect()
+  .then(() => {
+    // You are now connected to your Kuzzle instance.
+    return kuzzle.server.now();
+  })
+  .then(serverTime => console.log(serverTime))
+  .catch(error => console.error(error));
+
+// With async/await
```

Comment:
What's the purpose of proving an async/await example?

Reply:
Same reason as above